front/src/config/i18n/fr.json

```diff
@@ -243,6 +244,10 @@
         "addButton": "+",
         "substractButton": "-"
       },
+      "devices": {
+        "editDeviceFeaturesLabel": "Sélectionnez les appareils que vous voulez afficher ici. Seuls les capteurs / et interrupteurs on/off sont supportés.",
```

Comment:
are we only supporting sensors & on/off switch?

Reply:
IDK, If have copied the same behaviour as DevicesInRoom